app/commands/recent.py

```diff
@@ -107,3 +114,61 @@ async def recent(context: Context):
         embed=embed, reference=context.message, mention_author=True,
         view=replay
     )
+
+
+def get_difficulty_info(score: DBScore) -> Tuple[float, float, float]:
+    # fc_pp, star rating, approximate lazer sr (rx/ap)
+    beatmap_file = app.session.storage.get_beatmap(score.beatmap_id)
+
+    if not beatmap_file:
+        app.session.logger.error(
+            f'pp calculation failed: Beatmap file was not found! ({score.user_id})'
+        )
+        return 0.0, 0.0, 0.0
+
+    mods = Mods(score.mods)
+
+    if Mods.Nightcore in mods and not Mods.DoubleTime in mods:
+        # NC somehow only appears with DT enabled at the same time...?
```

Comment:
this is because of osu!stable

Reply:
Is there any reason for this btw.? I am still confused about that

Reply:
it's existed since always. pretty sure it was decided so that they wouldn't have to think about nightcore and rather just DT, since nightcore is just DT + pitch change